modules/custom/openy_hours_formatter/js/openy_hours_formatter.js

```diff
@@ -22,4 +23,82 @@
       });
     }
   };
+
+  Drupal.behaviors.today_hours = {
+
+    /**
+     * Used to track the setInveral.
+     */
+    refreshTimer: false,
+
+    /**
+     * See https://moment.github.io/luxon/docs/manual/formatting.
+     * @returns {string}
+     */
+    getDayOfWeek() {
+      return luxon.DateTime.local().setZone('America/Los_Angeles').toFormat('c');
+    },
+
+    /**
+     * See https://moment.github.io/luxon/docs/manual/formatting.
+     * @returns {string}
+     */
+    getDate() {
+      return luxon.DateTime.local().setZone('America/Los_Angeles').toFormat('yyyy-LL-dd');
```

Comment:
never set a specific timezone. Open Y is used across the USA and even in other countries. Also, I do not see any reason why `luxon` library is used here

Reply:
Yes the hard coded zone is an error.